tests/Utils/toChecksumAddress.test.ts

```diff
@@ -0,0 +1,27 @@
+import * as sinon from 'sinon';
+import Web3 = require('web3');
+
+import assert from '../test_utils/assert'
+import SpyAssert from '../test_utils/SpyAssert';
+import Utils from '../../src/Utils'
+
+describe('Utils.toChecksumAddress()', () => {
+  it('should be successfully', () => {
+    let address = '0x123Ad';
+
+    const web3Spy = sinon.stub(Web3.utils, 'toChecksumAddress').returns(address);
```

Comment:
It's a pure function call, Do we need stub?

Reply:
It is unit test and it has external dependency of `web3`
So i think we should stub the external dependencies.